modules/controlfreec/1.1/config/default.yaml

```diff
@@ -17,41 +17,53 @@ lcr-modules:
         options:
             configFile: "{MODSDIR}/config/freec/config_WGS.txt"
             BedGraphOutput: FALSE
-            breakPointThreshold: 0.8 #use 0.6 for more segments to get more CNVs
+            breakPointThreshold: 0.6 #use 0.6 for more segments to get more CNVs
             breakPointType: 2
```

Comment:
can you also please add comment here to show what 2 means, and if there are other options user can set?

Reply:
Agree with Kostia, there are a few other parameters that could use a comment so users know what the parameter does and if/when it should be changed.

Reply:
Okay!